integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/partition/TestShowPartitions.scala

```diff
@@ -150,6 +150,11 @@ class TestShowPartition  extends QueryTest with BeforeAndAfterAll {
 
   }
 
+  test("show partition table: desc formatted should show partition type"){
+    //check for partition type exist in desc formatted
+    checkExistence(sql("describe formatted hashTable"),true,"Partition Type")
```

Comment:
check the output of the type, whether it is correct

Reply:
@jackylk please review